src/pages/myGeotab/guides/codeBase/usingInDotnet.tsx

```diff
@@ -14,14 +16,14 @@ const nugetPackage: ReactNode = (
         <p>Please remember to regularly check for Geotab.Checkmate.ObjectModel nuget package updates. Ideally, your integration should use the same Geotab.Checkmate.ObjectModel nuget package version
             as the one that your database is on.</p>
         <InformationalBox>
-            <p>Quick start in <a href="../../api/clients">{/*TODO: fix link*/}API Clients</a></p>
+            <p>Quick start in <Link to="/myGeotab/apiClients">API Clients</Link></p>
```

Comment:
@kmets89 The current site has hash links for the Javascript & Java pages so that the link goes to the specific section of the API Clients page.
```suggestion
            <p>Quick start in <HashLink to="/myGeotab/apiClients#dotnet">API Clients</HashLink></p>
```